_docs_integrate/delete-identities.md

```diff
@@ -0,0 +1,113 @@
+{: .notice--warning}
+
+Regardless of whether an [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) is an App user or a Connector, it is stored on the Backbone.
+This guide describes the different [options for Identity deletion]({% link _docs_integrate/delete-identities.md %}#options-for-identity-deletion) from the Backbone.
+There is usually a predefined grace period in each process of Identity deletion, during which the Identity can cancel its deletion if it no longer wants to be deleted.
+Please note that an Identity can never trigger the process of deletion of another Identity.
+
+## IdentityDeletionProcesses
+
+From a technical perspective, the process of Identity deletion is described by a data object of type [IdentityDeletionProcess]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess). It can be uniquely identified by its `id`.
+An IdentityDeletionProcess can have `"WaitingForApproval"`, `"Rejected"`, `"Approved"` or `"Cancelled"` as its `status`.
+If it has `"WaitingForApproval"` or `"Approved"` as `status`, it is also referred to as an **active IdentityDeletionProcess**.
+There can be at most one active IdentityDeletionProcess per Identity.
+There are three [use cases]({% link _docs_integrate/use-cases.md %}) for getting one or more already existing [IdentityDeletionProcesses]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess):
+
+- If the `id` of an IdentityDeletionProcess is known, it can be viewed by calling the [Get IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-get-identitydeletionprocess.md %}) use case.
+- All IdentityDeletionProcesses of an Identity can be viewed by utilizing the [Get IdentityDeletionProcesses]({% link _docs_use-cases/use-case-transport-get-identitydeletionprocesses.md %}) use case. This includes IdentityDeletionProcesses with `"Cancelled"` or `"Rejected"` as `status` in particular.
+- The [Get active IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-get-active-identitydeletionprocess.md %}) use case can be executed to view the currently active IdentityDeletionProcess if one exists. If none exists, the [error code]({% link _docs_integrate/error-codes.md %}) `error.runtime.identityDeletionProcess.noActiveIdentityDeletionProcess` arises if an attempt is made to apply the use case anyway.
+
+## Options for Identity Deletion
+
+In principle, there are several options for deleting an Identity from the Backbone.
+For example, depending on the Backbone environment, it is conceivable to set up automatic Identity deletion after a long period of Identity inactivity.
+More essential are the two options for actively deciding to delete an Identity from the Backbone:
+
+- Initiation of the process of Identity deletion by the Identity itself.
+- The Backbone Admin UI triggers the process of Identity deletion and the Identity subsequently approves the triggered deletion of its Identity.
```

Comment:
Please don't talk about the `Backbone Admin UI` as an Identity, instead speak about the `Backbone Administrator`. I don't think we should mention the UI at all in this docs.

Reply:
Hm, I have to see how it's used already.